wealth-management-client/src/app/reports/userviews/userview-grid.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { MyStock } from '../../modal/mystock';
+import * as myStocks from '../../data/my-investment.json';
+
+@Component({
+  selector: 'userview-grid',
+  templateUrl: './userview-grid.component.html',
+  styleUrls: ['./userview-grid.component.css']
+})
+export class UserviewGridComponent {
+  userInvestmentList: MyStock[] = new Array<MyStock>();
+  scripts: string[] = new Array<string>();
+
+  constructor() {
+    this.getMyInvestmentList();
+  }
+
+  getMyInvestmentList() {
+    let myStocksData = JSON.parse(JSON.stringify(myStocks));
+    myStocksData.map(data => {
+        let mystock: MyStock = new MyStock();
```

Comment:
let stock: Stock = new Stock();
stock.isUserPortfolio = true;
stock.setMarketPrice(value.quoteSummary.result[0].price.regularMarketPrice.raw);
stock.setGridData(data);

Reply:
Get recommended data also while iterating over myStockList.

Reply:
Looks like the User story understanding is different between both of us. Lets have a call and sync-up
This check-in does not include all the functionality.
Did you check the TBD list in my comments?

Reply:
Done.